invenio_communities/assets/semantic-ui/js/invenio_communities/api/communityApi.js

```diff
@@ -0,0 +1,109 @@
+// This file is part of React-Invenio-Deposit
```

Comment:
I suggest to rename the file either to `community.js` or to be aligned with the name of the class it contains